client/src/components/NavBar/NotificationRequest/NotificationRequest.tsx

```diff
@@ -24,11 +24,11 @@ const NotificationRequest: React.FC<IProps> = ({
 
```

Comment:
Is `endOf('day')` necessary for line 23? Notification would display `in <x> hours` instead of `<x> hours ago` for notifications created on the same day.